version1/02 - Object types.md

```diff
@@ -480,6 +485,7 @@ The following notation is used in the tables below:
 |           | ref       | refAvskrivning                   |          |             |                 |  |      |      |   x   |      |  x   |        |                                                                                                                                                                                                                                                                                                        |
 |           | ref       | refTilknyttetAvskrivning         |          |             |                 |  |      |      |   x   |      |  x   |   x    |                                                                                                                                                                                                                                                                                                        |
 |           | ref       | refMerknad                       |          |             |                 |  |      |      |   x   |      |  x   |   x    |                                                                                                                                                                                                                                                                                                        |
+|           | ref       | refNasjonalIdentifikator         |          |             |                 |  |      |      |   x   |      |  x   |   x    |                                                                                                                                                                                                                                                                                                        |
```

Comment:
We cannot unlink the national identifier from the registry entry. We also have mistakenly marked that we can unlink notes and sign-offs above.
```suggestion
|           | ref       | refNasjonalIdentifikator         |          |             |                 |  |      |      |   x   |      |  x   |        |                                                                                                                                                                                                                                                                                                        |
```